### ISSUE TITLE
Loss Behavior in LLM-based TTS Training

### ISSUE THREAD

**Issue by gevmin94** [zhenye234/LLaSA_training#52]
Hi,

We’re currently experimenting with an LLM-based TTS system and are adapting the LLM component using Llama 3-1B-Instruct. One behavior we’ve noticed is that the loss initially rises during the first few thousand steps before it starts to decrease.

<img width="963" height="454" alt="Image" src="https://github.com/user-attachments/assets/62b8dc14-a218-4f61-b1e9-55548a5e7118" />

We’ve tried different hyperparameters, optimizers, and schedulers, but this pattern persists. Since I know you’ve worked on similar setups, I’d love to hear about your experience with this issue.
